### ISSUE TITLE
Trinity Dataset Pretrained Feature Extractor for Gesture Perceptual Loss 

### ISSUE THREAD

**Issue by anithselva** [yesheng-THU/GFGE#3]
Hello,

Would it be possible to have a Trinity dataset pre-trained feature extractor for the GP Loss?
Training the Feature Extractor on this dataset from scratch looks like it requires some changes to the repo code, and it's taking me some time to get familiar with it to be confident in these changes. 

Hoping to calculate eval metrics (i.e. FGD) on Trinity but as you mentioned the feature extractor was trained on TED Dataset. If you can provide with the pretrained model for Trinity that would be greatly appreciated

Thanks
